modules/check_in/spec/services/v2/chip/service_spec.rb

```diff
@@ -655,4 +655,43 @@
       end
     end
   end
+
+  describe '#travel_params' do
+    let(:params_without_travel_attributes) { { appointment_ien: 'test-appt-ien' } }
+    let(:params_with_travel_attributes) do
+      {
+        appointment_ien: 'test-appt-ien',
+        is_travel_enabled: true,
+        travel_submitted: false
+      }
+    end
+
+    context 'when called without travel parameters' do
+      let(:hsh_without_travel_attributes) do
```

Comment:
should this spec cover when travel parameters are not sent at all? would those params still be nil?